proposals/umoci.md

```diff
@@ -0,0 +1,108 @@
+
+### Governance and Releases ###
+This project would incorporate the Governance and Releases processes from the OCI project template: https://github.com/opencontainers/project-template.
+
+It should be noted that since umoci is not a specification, it is not bound by the ordinary quorum and voting rules for specification release.
+As such, new versions will be released as regularly as needed without the need for a quorum vote.
+This is to avoid the serious delays in releases encountered within the `runc` project.
+
+Until there are enough additional maintainers, PRs will be merged with only one maintainer LGTM required.
+Maintainers are encouraged to not merge their own PRs immediately (preferably waiting at least 24 to 48 hours to allow another maintainer to review it), but this is not a strict rule.
+
+### Project Communications ###
+The proposed project would continue to use existing channels in use by the OCI developer community for communication including:
+
+ * GitHub for issues and pull requests.
+ * The [`dev@opencontainers.org`][oci-ml] email list.
+ * The weekly OCI developer community conference call.
+ * The `#opencontainers` IRC channel on Freenode.
+ * The [OCI Slack workspace][oci-slack].
+ * The [OCI Matrix Room][oci-matrix].
+
+[oci-ml]: mailto:dev@opencontainers.org
+[oci-slack]: https://opencontainers.slack.com/
+[oci-matrix]: https://matrix.to/#/#opencontainers:matrix.org
+
+### Roadmap ###
+umoci still has a fairly large scope of work, and this work would continue under OCI.
+Examples of such work include:
+
+ * Hookable execution to allow for custom storage drivers to be defined by users (to avoid the maintainence overhead of supporting such drivers within umoci).
+ * Security hardening by porting umoci to libpathrs.
+ * Re-thinking the method by which images are referenced in the command-line.
+ * Expanded support for configuring descriptor annotations in arbitrarily-structured OCI metadata trees.
+ * Committing to a stable Go API and releasing a 1.0 of the project.
+ * Experimenal design work for a new OCI image-spec layer format (code-named "OCIv2").
+
+Additionally, merging the validation code into umoci (either as a sub-project, or as a sub-command of the main `umoci` tool) would be work that should be carried out as soon as this proposal is actioned.
+
+## Frequently Asked Questions (FAQ)
+*Q: Does this change the OCI Charter or Scope Table?*
+A: Yes, this will add an additional row to the [OCI Scope Table][oci-scope-table], which does not currently include any mention of generic OCI image tooling.
+We feel that this is a natural extension of the OCI project's goals, and is very similar to `runc` in its justification for being included in the scope of OCI.
+This proposal does not intend to amend the [OCI Charter][oci-charter].
+
+[oci-scope-table]: https://www.opencontainers.org/about/oci-scope-table
+[oci-charter]: https://www.opencontainers.org/about/governance
```

Comment:
The link is 404. I think it is ok to link to file in repo

Reply:
Yup, already working on it (I also need to remove references to the OCI scope table).